labonneboite/web/admin/views/office_admin_update.py

```diff
@@ -10,6 +10,14 @@
 from labonneboite.web.admin.utils import datetime_format, AdminModelViewMixin
 from labonneboite.conf import settings
 
+def is_siret(siret):
+    # A valid SIRET is composed by 14 digits
+    try:
+        int(siret)
+    except ValueError:
+        return False
+
+    return len(siret) == 14
```

Comment:
stp peux tu factoriser cela dans `common/util.py` ou encore mieux `common/siret.py` - j'aurai juré qu'on avait déjà une telle méthode mais impossible de la retrouver j'ai du rêver

par contre il y a bel et bien un autre endroit où factoriser => admin/forms.py/siret_validator 

DNRY for the win